src/sentry/templates/sentry/emails/activity/generic.txt

```diff
@@ -7,7 +7,7 @@
 
 ## Issue Details
 
-{{ group.message_short }}
+{{ group.title }}
```

Comment:
Shouldn't this become `group.message` now?

Reply:
to quote the existing code:

```
    @property
    def message_short(self):
        warnings.warn('Group.message_short is deprecated, use Group.title', DeprecationWarning)
        return self.title
```